src/main/java/cs2103/duke/ToDo.java

```diff
@@ -0,0 +1,20 @@
+package cs2103.duke;
+
+/**
+ * This class encapsulates a Todo Task object, which inherits from the Task class.
+ */
+public class ToDo extends Task {
+    protected int index;
```

Comment:
Perhaps you could consider renaming `index` such that it is more self-explanatory about what it is referring to.